Experimental/MultiplayerUseCases/Assets/Common/Scripts/NetworkManagerUI.cs

```diff
@@ -87,25 +118,31 @@ void OnSceneLoaded(Scene loadedScene, LoadSceneMode arg1)
             }
         }
 
-        void EnableAndHighlightButtons(Button buttonToHighlight, bool enable)
```

Comment:
Seeing comments above: can you re-implement this method and put the code that manages the state of buttons here? It did what you're trying to do, but for UGUI. You just need to adapt its code to UITK.

Reply:
I put the function back in now, the one thing I am struggling with right now is bringing back the hovering functionality

I tried it with:
`m_Buttons.Where(button => button.enabledInHierarchy).ToList().ForEach(button =>
            {
                button.RegisterCallback<MouseEnterEvent>(evt => button.style.backgroundColor = m_ButtonHoverColor);
                button.RegisterCallback<MouseLeaveEvent>(evt => button.style.backgroundColor = m_ButtonColor);
            });
            if (m_DisconnectButton.enabledInHierarchy)
            {
                m_DisconnectButton.RegisterCallback<MouseEnterEvent>(evt => m_DisconnectButton.style.backgroundColor = m_ButtonHoverColor);
                m_DisconnectButton.RegisterCallback<MouseLeaveEvent>(evt => m_DisconnectButton.style.backgroundColor = m_ButtonColor);`
                
   but this brings back the hovering backgroundcolor for disabled buttons as well for some reason